tags/outdated.tag

```diff
@@ -1,5 +1,2 @@
-type: text
-
----
-
-Please read this when considering what Minecraft version to use for your server: https://matthewmiller.dev/blog/which-minecraft-version/
\ No newline at end of file
+type: alias
+target: outdatedclient
```

Comment:
This no longer works as it used to, this needs to be looked at more.

Reply:
?